scripts/check_unit.sh

```diff
@@ -12,4 +12,11 @@ echo "Running $0"
 PKGS=`go list github.com/hyperledger/aries-framework-go/... 2> /dev/null | \
                                                  grep -v /mocks`
 
+CWD=$PWD
+if [[ ! -d "pkg/didcomm/transport/http/testdata" ]]; then
```

Comment:
do we need to add every package that use go generate here ?

Reply:
use go generate ./...

Reply:
done